data/json/effects.json

```diff
@@ -1354,6 +1359,7 @@
             "Heavy Asthma",
             "Heavy Asthma"
         ],
+        "desc": [ "You suffer from respiratory disorder that makes breathing difficult.\nAn asthma attack can occur from time to time so have your inhaler ready." ],
```

Comment:
Recommend `You suffer from a respiratory` ...
Second line, `time to time, so have your inhaler nearby.`